### PR TITLE
Added ability to list servers/nodes.

### PR DESCRIPTION
**Pull request by aimonb**
Added ability to list servers/nodes.

### CODE REVIEW

**Comment by jr2730** [comment]
I don't see much GET support. Is get supposed to be different than list?
If not, maybe make get an alias for list?
If so, I don't see any GET support.

**Comment by relaxdiego** [comment]
I can't think of a scenario when anyone would need a separate GET in this case. Ansible always works with a list of hosts even if the list only contains one host.

**Comment by aimonb** [comment]
get is alias for list.

**Comment by jr2730** [comment]
Might be better to use the AnsibleModule "alias" capability.
Cause right now you have 2 different "commands" that are both treated the same.
And it makes the documentation more confusing (UNLESS you somehow plan on separating the 2 commands at some point).

**Comment by aimonb** [comment]
As far as I know alias only works for arguments, not their values.

**Comment by aimonb** [comment]
I am thinking we just drop 'get'?

**Comment by aimonb** [comment]
'get' has been removed.

File: cloud/dimensiondata/dimensiondata_compute.py [hunk 1]
```diff
@@ -601,6 +607,19 @@ def core(module):
...
+        for node in nodes:
+            nodes_list.append(node_to_node_obj(node))
+        module.exit_json(changed=False,
+                         msg="Successfully retrieved node list.",
+                         nodes=nodes_list)
+
+    # ensure not list/get so we acquiesce
+    if module.params['nodes'] is None:
+        module.fail_json(msg="'nodes' argument must be specified when 'ensure'"
+                             " is not list/get.")
```
Comment (relaxdiego) [inline cloud/dimensiondata/dimensiondata_compute.py]: you missed a spot ("list/get") :-)

**Review by relaxdiego — CHANGES_REQUESTED** [review]

File: cloud/dimensiondata/dimensiondata_compute.py (hunk 1 above)
Comment (aimonb) [reply cloud/dimensiondata/dimensiondata_compute.py]: doh! thanks!

**Review by relaxdiego — APPROVED** [review]

**Pull request merged**